llvm/include/llvm/Support/AllocToken.h

```diff
@@ -49,9 +49,9 @@ struct AllocTokenMetadata {
 /// \param Metadata The metadata about the allocation.
 /// \param MaxTokens The maximum number of tokens (must not be 0)
 /// \return The calculated allocation token ID, or std::nullopt.
-std::optional<uint64_t> getAllocTokenHash(AllocTokenMode Mode,
-                                          const AllocTokenMetadata &Metadata,
-                                          uint64_t MaxTokens);
+LLVM_ABI std::optional<uint64_t>
+getAllocToken(AllocTokenMode Mode, const AllocTokenMetadata &Metadata,
```

Comment:
Why ABI?

Reply:
We should export it, so other language implementations can use it as well.

Reply:
Per this https://llvm.org/docs/InterfaceExportAnnotations.html
afaik we should put LLVM_ABI, so it works consistently. I think the argument is that we just don't know how the LLVM library is linked.